src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOUR_INDEX = 2;
+    private static final int INCOME_INDEX = 3;
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        try {
+            LocalDate startDate = LocalDate.parse(dateFrom, FORMATTER);
+            LocalDate endDate = LocalDate.parse(dateTo, FORMATTER);
+            StringBuilder stringBuilder = new StringBuilder();
+            stringBuilder.append("Report for period ")
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
            stringBuilder.append(HEADER)
```